components/date-time/style.scss

```diff
@@ -43,6 +43,7 @@ $datepicker__border-color: $light-gray-500;
 	align-items: center;
 	justify-content: center;
 	margin-top: 10px;
+	max-width: 248px;
```

Comment:
Why was this size picked? Is it the max-width of the datepicker itself? If so, we should use a variable.

Reply:
It's just a (manually calculated) width of the calendar itself, which doesn't seem to change at all. The number isn't used elsewhere, so whilst we *could* add a variable for it, I'm not sure we'd ever need/want to use it elsewhere.

Reply:
Makes sense; a comment explaining that would be great, in that case.